<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TCPServer</class>
 <widget class="QWidget" name="TCPServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TCPServer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>TCPServer</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>服务端配置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>IP地址</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_ServerAddress">
           <property name="text">
            <string>127.0.0.1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>端口号</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QSpinBox" name="spinBox_ServerPort">
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="value">
            <number>4800</number>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="pushButton_ServerControl">
           <property name="text">
            <string>开始监听</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>接收显示</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QTextBrowser" name="textBrowser"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>回复配置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0" rowspan="2">
          <widget class="QTextEdit" name="textEdit_ResponseMessage"/>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="checkBox_EnableAutoEcho">
           <property name="text">
            <string>自动回复(回显)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_SendResponseMessage">
           <property name="text">
            <string>回复</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>目标节点</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_ClientConnected">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
